plugins/typescript/.toolkitrc.yml

```diff
@@ -0,0 +1,4 @@
+hooks:
+  'build:local': TypescriptBuild
+  'run:local': TypeScriptWatch
```

Comment:
**question:** Does `TypeScriptWatch` need to run before any other plugin such as `nodemon`? If so, we should mention that in the readme.

Reply:
I've updated the task so that it can run before or after other `run:local` tasks as none of them should be blocking